### DISCUSSION TITLE
Migrating to sanoid from a homebrew periodic snapshot solution?

### DISCUSSION THREAD

**Discussion by tmladek** [jimsalterjrs/sanoid#800] [Q&A] [answered]
Hi, I'm managing a semi-public (community) NAS, so zfs's snapshots are a very welcome feature. Initially, I just had a periodic `systemd` service take a snapshot of the relevant datasets, then of course after some time I realized the need to prune the old & irrelevant ones, so I wrote a script to delete old snapshot on a staggered basis (keep 2 monthly snapshots, 3 weekly, etc).

It works, but it's not a solution I'm happy with, and I'd like to migrate to a proper solution - like sanoid!

How would I go about this (if it's possible)? Does sanoid just look at the snapshot timestamp, so I could just set it up and let it take over with no "migration" needed? Is there a naming scheme that I should rename my existing snapshots to? Or does it keep state somewhere completely different?

Thanks in advance!

**Comment by jimsalterjrs** [comment, marked as answer]
Naming scheme. As long as your snapshots are named in sanoid's format, it 
will treat them as its own and manage them according to policy.
